src/table/table.scss

```diff
@@ -4,9 +4,8 @@
 @import '../icon/index';
 @import './variables';
 
-@mixin iui-tables-table {
+@mixin iui-table {
   @include iui-reset;
-
   @include iui-font-family;
   display: flex;
   flex-direction: column;
```

Comment:
padding is not needed here, as reset brings padding: 0 already..

Reply:
Good catch, removed the padding.